meta-resin-common/classes/binary-compress.bbclass

```diff
@@ -1,7 +1,26 @@
-# e.g.  FILES_COMPRESS = "/path/bin1 /path/bin2 /path/bin3"
+# Copyright 2016-2017 Resinio Ltd.
```

Comment:
I think only 2016 (the oldest date) should be applied here, otherwise I worry that we're saying that we only protect this file up to 2017.